nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -326,6 +326,9 @@ public class NiFiProperties extends ApplicationProperties {
     public static final String PYTHON_CONTROLLER_DEBUGPY_HOST = "nifi.python.controller.debugpy.host";
     public static final String PYTHON_CONTROLLER_DEBUGPY_LOGS_DIR = "nifi.python.controller.debugpy.logs.directory";
 
+    // kubernetes properties
+    public static final String KUBERNETES_RESOURCE_PREFIX = "nifi.kubernetes.resource.prefix";
```

Comment:
Recommend naming this more specifically for clarity:
```suggestion
    public static final String CLUSTER_LEADER_ELECTION_KUBERNETES_LEASE_PREFIX = "nifi.cluster.leader.election.kubernetes.lease.prefix";
```